packages/ketcher-core/src/application/editor/actions/rotate.ts

```diff
@@ -34,26 +34,37 @@ export function fromFlip(restruct, selection, dir, center) {
 
   const action = new Action()
 
-  if (!selection) selection = structSelection(struct)
+  if (!selection) {
+    selection = structSelection(struct)
+  }
```

Comment:
Why is this needed?

Reply:
I think it's much more readable, can't miss the brackets :)